jupyter-extension/jupyterlab_pachyderm/pfs_manager.py

```diff
@@ -469,12 +474,62 @@ def current_datum(self) -> dict:
             all_datums_received=True,
         )
 
-    def _get_fileinfo_path(self, fileinfo: pfs.FileInfo) -> Path:
+    def download(self):
```

Comment:
Could you give this method a docstring outlining what its doing. It looks like it is downloading files to a local cache directory and then to linking that directory from /pfs.